src/chart/boxplot/BoxplotView.ts

```diff
@@ -180,17 +183,76 @@ function updateNormalBoxData(
 
     el.useStyle(data.getItemVisual(dataIndex, 'style'));
     el.style.strokeNoScale = true;
-
     el.z2 = 100;
 
     const itemModel = data.getItemModel<BoxplotDataItemOption>(dataIndex);
     const emphasisModel = itemModel.getModel('emphasis');
 
+    if (seriesModel.option.label.show) {
```

Comment:
We don't access the attributes using `seriesModel.option`. Please use `const labelModel = seriesModel.get('label'); const show = labelModel.get('show');` instead.

Please also change `seriesModel.option` in other places.